doc/designs/quic-design/rx-depacketizer.md

```diff
@@ -0,0 +1,194 @@
+RX depacketizer
+===============
+
+This component takes a QUIC packet and parses the frames contained therein,
+to be forwarded to appropriate other components for further processing.
+
+In the [overview], this is called the "RX Frame Handler".  The name "RX
+depacketizer" was chosen to reflect the kinship with the [TX packetizer].
+
+Structures
+----------
+
+### Connection
+
+Represented by an `QUIC_CONNECTION` object, defined in
+[SSL object refactoring using SSL_CONNECTION object].
+
+### Stream
+
+Represented by an SSL object.
```

Comment:
Externally yes. But internally I expect something like `OSSL_QUIC_STREAM`

Reply:
Side note on names: we have `QUIC_CONNECTION`, but `OSSL_QUIC_STREAM`?  Can we *please* decide on the prefixing?

Reply:
Is `QUIC_STREAM` defined in any PR yet?  Just so I know what to refer to...

Reply:
Fixed (with no reference to its definition for the time being)